docs/docs/nlu-training-data.mdx

```diff
@@ -168,3 +168,41 @@ For example, you should not only have example like `fly FROM x TO y`, but also i
 
 To fill slots from entities with a specific role/group, you need to either define a custom slot mappings using
 [Forms](forms.mdx) or use [Custom Actions](custom-actions.mdx) to extract the corresponding entity directly from the tracker.
+
+
+### Entity with BILOU tagging
```

Comment:
Placement is fine for now, i'll find a spot with it when reviewing your updates to this page :) 
```suggestion
### BILOU Entity Tagging
```